internal/ackhandler/sent_packet_handler.go

```diff
@@ -49,8 +49,12 @@ type sentPacketHandler struct {
 	handshakePackets *packetNumberSpace
 	appDataPackets   *packetNumberSpace
 
-	peerCompletedAddressValidation bool
-	handshakeComplete              bool
+	peerCompletedAddressValidation bool // always true for the server
```

Comment:
Can you make the difference between `peerAddressValidated` and `peerCompletedAddressValidation` clearer, either through better naming or documentation?

Reply:
Done.